src/Controller/Api/Carrinho/ConsultarCarrinhoController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Controller\Api\Carrinho;
+
+use App\Service\BuscarTodosCarrinhoService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ConsultarCarrinhoController extends AbstractController
+{
+  public function __construct(
+    private BuscarTodosCarrinhoService $buscarTodosCarrinhoService,
+  ){}
+
+  #[Route('/api/carrinho', name: 'consultar_carrinho', methods: ['GET'])]
+  public function index(): JsonResponse
+  {
+    try {
+      $carrinhos = $this->buscarTodosCarrinhoService->execute();
```

Comment:
da para fazer assim também, fica menos código!

```suggestion
  #[Route('/api/carrinho', name: 'consultar_carrinho', methods: ['GET'])]
  public function index(BuscarTodosCarrinhoService $buscarTodosCarrinhoService): JsonResponse
  {
    try {
      $carrinhos = $buscarTodosCarrinhoService->execute();
```